api/app/serializers/v1/entitlement_serializer.rb

```diff
@@ -4,44 +4,43 @@ class EntitlementSerializer < ManifoldSerializer
 
     ID = Types::Serializer::ID.meta(read_only: true)
 
-    CURRENT_STATE = Types::Coercible::String.enum(EntitlementState.map(&:to_s)).meta(example: "active", read_only: true)
+    ENTITLEMENT_STATE_OPTIONS = EntitlementState.map(&:to_s)
+    CURRENT_STATE = Types::Coercible::String.enum(*ENTITLEMENT_STATE_OPTIONS).meta(example: "active", read_only: true)
```

Comment:
I changed this because the enum method takes several parameters rather than an array of strings.

Reply:
Thank you for catching that!